### ISSUE TITLE
Fails to load on SBCL 1.4.3, under threat of removal from QL

### ISSUE THREAD

**Issue by vityok** [melisgl/mgl-pax#5]
As you might already know, there is a problem with mgl-pax that puts it under [threat of being deleted](http://blog.quicklisp.org/2018/01/build-failures-with-asdf-331.html) from Quicklisp.

When loading on SBCL 1.4.3 the following error is reported:

```
[package mgl-pax]....................; 
; caught ERROR:
;   READ error during COMPILE-FILE:
;   
;     The symbol "PRIMARY-SYSTEM-NAME" is not external in the ASDF/FIND-SYSTEM package.
;   
;       Line: 2631, Column: 55, File-Position: 109423
;   
;       Stream: #<SB-INT:FORM-TRACKING-STREAM for "file /home/user/quicklisp/dists/quicklisp/software/mgl-pax-20151218-git/src/pax.lisp" {1008B889D3}>

debugger invoked on a UIOP/LISP-BUILD:COMPILE-FILE-ERROR in thread
#<THREAD "main thread" RUNNING {1001978083}>:
  COMPILE-FILE-ERROR while compiling #<CL-SOURCE-FILE "mgl-pax" "src" "pax">

Type HELP for debugger help, or (SB-EXT:EXIT) to exit from SBCL.

```

The cl-string-match library I'm developing depends on mgl-pax is therefore fails to load too and is under threat of a cascading removal.

With best regards,
Victor

**Comment by vityok** [comment]
There is even a pull request already to fix this issue:  #3.

**Comment by vityok** [comment]
Fixed by commit 53f7aa6a56d173ef3d8a2527506a7b339018db6c or pull request #3.
